### PR TITLE
update testing fake api

### PR DESCRIPTION
**Pull request by trevorrjohn**

### CODE REVIEW

File: tests/dummy/snippets/testing.js [hunk 1]
```diff
@@ -1,19 +1,64 @@
...
-    toggleFeed() {},
-    showInAppMessage() {},
+// Use this as a fake version of the Appboy Api
+const appboyApi = {
+  initialize(apiKey, options) {
+  },
+  changeUser(userId) {
+  },
+  destroy() {
+  },
```
Comment (blimmer) [inline tests/dummy/snippets/testing.js]: Total nitpick, but could you match the style of the rest of this?

``` javascript
destroy() { },
```

**Review by blimmer — CHANGES_REQUESTED** [review]
Wow, definitely a lot of methods, but helpful for copy / paste for others.

**Comment by trevorrjohn** [comment]
ok should be updated now.

**Review by blimmer — APPROVED** [review]

**Comment by blimmer** [comment]
thanks! i'll merge this once tests pass.

**Pull request merged**